src/containerapp/azext_containerapp/_compose_utils.py

```diff
@@ -47,3 +49,14 @@ def validate_memory_and_cpu_setting(cpu, memory, managed_environment):
         logger.warning(  # pylint: disable=W1203
             f"Invalid CPU reservation request of {cpu}. The default resource values will be used.")
     return (None, None)
+
+
+def resolve_gpu_configuration_from_service(service):
```

Comment:
Please add test for `az containerapp compose crease` due to this PR add support gpu param in this command